tests/instruments/smaract/test_scu_ascii.py

```
@@ -0,0 +1,250 @@
+    ):
+        pass  # verify the expected communication.
+
+
+def test_close():
+    with expected_protocol(
+            SmarActSCULinear,
+            [(':GID', None)],  # no communication expected
+    ) as inst:
+        inst.close()
+        # verify instrument is no longer usable
+        with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)